test/libyul/yulOptimizerTests/controlFlowSimplifier/terminating_for_revert_plus_break.yul

```diff
@@ -0,0 +1,20 @@
+{
+	for {} calldatasize() { mstore(1, 2) } {
+		let x := 7
+		mstore(4, 5)
+		break
+		revert(0, x)
+	}
+}
+// ====
+// step: controlFlowSimplifier
+// ----
+// {
```

Comment:
Why is this case not simplified?

Reply:
Because there is a `break` in the code. If you combine it with `DeadCodeEliminator`, it will simplify.